src/slider/SliderFilledTrack.ts

```diff
@@ -4,21 +4,23 @@
  * We improved the hook [useSlider](https://github.com/chakra-ui/chakra-ui/blob/af613020125265914a9dcb74c92a07a16aa4ff8e/packages/slider/src/use-slider.ts)
  * to work with Reakit System
  */
-import { createComponent, createHook } from "reakit-system";
 import { BoxHTMLProps, useBox } from "reakit";
+import { createComponent, createHook } from "reakit-system";
 
 import { UseSliderReturn } from "./SliderState";
 import { SLIDER_FILLED_TRACK_KEYS } from "./__keys";
 
 export const useSliderFilledTrack = createHook<UseSliderReturn, BoxHTMLProps>({
-  name: "useSliderFilledTrack",
+  name: "SliderFilledTrack",
+  compose: useBox,
   keys: SLIDER_FILLED_TRACK_KEYS,
-  compose: [useBox],
 
   useProps(options, { style: htmlStyle, ...htmlProps }) {
+    const { styles } = options;
+
     return {
-      style: { ...options.styles.innerTrackStyle, ...htmlStyle },
       ...htmlProps,
+      style: { ...htmlStyle, ...styles.innerTrackStyle },
```

Comment:
Changed the order to enforce the applied styles?

Reply:
I wantedly changed the order, because these styles are necessary so the users don't override it with thier styles. 

What do you think?

Reply:
Yup thats totally fine, i'm just curious.